Package.swift

```diff
@@ -27,19 +27,19 @@ let package = Package(
         .library(name: "RxMoya", targets: ["RxMoya"])
     ],
     dependencies: [
-        .package(url: "https://github.com/Alamofire/Alamofire.git", .upToNextMajor(from: "5.4.3")),
-        .package(url: "https://github.com/ReactiveCocoa/ReactiveSwift.git", .upToNextMajor(from: "6.6.1")),
-        .package(url: "https://github.com/ReactiveX/RxSwift.git", .upToNextMajor(from: "6.2.0")),
-        .package(url: "https://github.com/Quick/Quick.git", .upToNextMajor(from: "4.0.0")), // dev
-        .package(url: "https://github.com/Quick/Nimble.git", .upToNextMajor(from: "9.2.0")), // dev
-        .package(url: "https://github.com/AliSoftware/OHHTTPStubs.git", .upToNextMajor(from: "9.0.0")) // dev
+        .package(url: "https://github.com/Alamofire/Alamofire.git", from: "5.4.3"),
+        .package(url: "https://github.com/ReactiveCocoa/ReactiveSwift.git", from: "6.6.1"),
+        .package(url: "https://github.com/ReactiveX/RxSwift.git", from: "6.2.0"),
```

Comment:
I think it's better to specify the same major version as podspec than to use the fixed version.
Is there a reason why you chose a fixed version?

Reply:
the `from: "semvar"` is an alias for up to the next major version. so selecting `Alamofire.git", from: "5.4.3"` allows any version from 5.x.x

we should definitely match what the Podspec & Cartfile have, I wish there was one place we could define all of those minimum versions to stay consistent. 

are the PodSpec/Cartfile significantly different?

Reply:
Is `from: "5.4.3"` correct to include all 5.x.x?
It seems confusing to me 
because the document saying that it starting with the minimum version going up to the next major version.

In the podspec file it is written as `~> 5.0`
how about 5.0.0 in the Package.swift file? (`from: "5.0.0"` or `.upToNextMajor(from: "5.0.0")`)

Reply:
ahh you're right, its technically 5.4.3 until 6.0.0, not including 6.0.0.

so 5.4.2 or below aren't acceptable

yeah, it is a little confusing, they're updating the SPM syntax very often

Reply:
Then, I think it would be good to change like podspec. (Alamofire, RxSwift etc..)
podspec : ~> 5.0 (`from: "5.0.0"` or .`upToNextMajor(from: "5.0.0")`)
What do you think about it?

Reply:
#2207